packages/now-cli/src/util/deploy/get-app-name.ts

```diff
@@ -1,12 +1,12 @@
 import path from 'path';
 import { NowError } from '../now-error';
 import { Output } from '../output';
-import { Config } from '../../types';
+import { NowConfig } from '../dev/types';
```

Comment:
If we're going to be using this type in places outside of `now dev`, then I think we should move the definition to `src/util/types.ts`.

Reply:
I was going to do a follow up PR to move it into a separate package completely such as `@now/build-utils` because we need it for backend APIs as well.